cypress/support/commands.js

```diff
@@ -0,0 +1,25 @@
+// ***********************************************
```

Comment:
Will we be using this file? i am not very experienced with cypress so I see a commented out file, I question it. :p hehehe

Reply:
yes, we will use this file, just not yet.